botr.py

```diff
@@ -25,51 +25,52 @@ def get_prefix(bot, message):
         for extension in initial_extensions:
             try:
                 bot.load_extension(extension)
+                print(extension +' has been loaded!')
```

Comment:
minor consistency thing, this should be a format string too like L30
```suggestion
                print(f'{extension} has been loaded!')
```

Reply:
Yeah thats old code i never bothered fixing since it did the job, agreed.